backtrader_plotting/bokeh/figureenvelope.py

```diff
@@ -108,26 +108,107 @@ def apply_hovertips(self, figures: List['FigureEnvelope']) -> None:
+        # "*"	m14	star
+        '*': ('asterisk', ["color", "size"], {}, {}),
+        # "h"	m15	hexagon1
+        'h': ('hex', ["color", "size"], {}, {}),
+        # "H"	m16	hexagon2
+        'H': ('hex', ["color", "size"],
+              {"angle": 45, "angle_units": "deg"}, {}),
+        # "+"	m17	plus
+        '+': ('plus', ["color", "size"], {}, {}),
+        # "x"	m18	x
+        'x': ('x', ["color", "size"], {}, {}),
+        # "X"	m24	x(filled)
+        'X': ('x', ["color", "size"], {}, {"size": -2}),
+        # "D"	m19	diamond
+        'D': ('diamond_cross', ["color", "size"], {}, {}),
+        # "d"	m20	thin_diamond
+        'd': ('diamond', ["color", "size"], {}, {}),
+        # "|"	m21	vline
+        '|': ('vbar', ["color"], {}, {}),
+        # "_"	m22	hline
+        '_': ('hbar', ["color"], {}, {}),
+        # 0 (TICKLEFT)	m25	tickleft
+        0: ('triangle', ["color", "size"],
+            {"angle": -90, "angle_units": "deg"}, {"size": -2}),
+        # 1 (TICKRIGHT)	m26	tickright
+        1: ('triangle', ["color", "size"],
+            {"angle": 90, "angle_units": "deg"}, {"size": -2}),
+        # 2 (TICKUP)	m27	tickup
+        2: ('triangle', ["color", "size"], {}, {"size": -2}),
+        # 3 (TICKDOWN)	m28	tickdown
+        3: ('triangle', ["color", "size"],
+            {"angle": 180, "angle_units": "deg"}, {"size": -2}),
+        # 4 (CARETLEFT)	m29	caretleft
+        4: ('triangle', ["fill_color", "color", "size"],
+            {"angle": -90, "angle_units": "deg"}, {}),
+        # 5 (CARETRIGHT)	m30	caretright
+        5: ('triangle', ["fill_color", "color", "size"],
+            {"angle": 90, "angle_units": "deg"}, {}),
+        # 6 (CARETUP)	m31	caretup
+        6: ('triangle', ["fill_color", "color", "size"], {}, {}),
+        # 7 (CARETDOWN)	m32	caretdown
+        7: ('triangle', ["fill_color", "color", "size"],
+            {"angle": 180, "angle_units": "deg"}, {}),
+        # 8 (CARETLEFTBASE)	m33	caretleft(centered at base)
+        8: ('triangle', ["fill_color", "color", "size"],
+            {"angle": -90, "angle_units": "deg"}, {"x": 0.25}),
+        # 9 (CARETRIGHTBASE)	m34	caretright(centered at base)
+        9: ('triangle', ["fill_color", "color", "size"],
+            {"angle": 90, "angle_units": "deg"}, {"x": -0.25}),
+        # 10 (CARETUPBASE)	m35	caretup(centered at base)
+        10: ('triangle', ["fill_color", "color", "size"],
+             {}, {"y": -0.25}),
+        # 11 (CARETDOWNBASE)	m36	caretdown(centered at base)
+        11: ('triangle', ["fill_color", "color", "size"],
+             {"angle": 180, "angle_units": "deg"}, {"y": 0.25}),
+        # "None", " " or ""	 	nothing
+        '': ('text', ["text_color", "text_font_size", "text"],
+             {}, {}),
+        ' ': ('text', ["text_color", "text_font_size", "text"],
+              {}, {}),
+        # '$...$' text
+        '$': ('text', ["text_color", "text_font_size", "text"],
+              {}, {}),
+    }
```

Comment:
https://github.com/verybadsoldier/backtrader_plotting/blob/master/backtrader_plotting/bokeh/utils.py

it could go in that file together with a function 
```
def get_marker_info(marker):
...
```
which would be called like this:
`fnc_name, attrs, vals, updates = get_marker_info(marker)`

Reply:
No need for a function in my opinion. Just put the table into a module and import it.